src/main/java/crawlercommons/sitemaps/extension/LinkAttributes.java

```diff
@@ -32,6 +34,10 @@
  * you to check for mistakes.</blockquote>
  */
 public class LinkAttributes extends ExtensionMetadata {
+
+    public static final String HREF = "href";
+    public static final String PARAMS = "params.%s";
```

Comment:
My only comment here is that `PARAMS` is a bit different from the other keys. Not a big deal, but I'd probably call it `PARAMS_PREFIX`, and make its value just `params.`

In any case I'd add a comment explaining how it's used for the map key.

Reply:
Thinking about PARAMS_PREFIX a little more, does it make sense to decrease the visiblity?  I'm not sure how useful it is outside of the class.